vuepress/docs/.vuepress/next.js

```diff
@@ -121,188 +173,232 @@ module.exports = {
                     {
                         title: 'Code of Conduct',
                         path: path + 'community/code-of-conduct.md'
-                    },
+                    }
                 ]
             },
             {
                 title: 'Release Notes',
-                path: path  + 'releases/'
+                children: [
+                    {
+                        title: 'New Features and Major Fixes',
+                        path: path + 'releases/'
+                    },
+                    {
+                        title: 'Previous Releases',
+                        path: path + 'releases/#previous-realeases',
+                    }
+                ]
             },
         ]
     },
 
     tutorialsSidebar: function(path) {
         return [
             {
-                title: 'Overview',
-                path: path,
-            },
-            {
-                title: 'Micro Frontends',
-                path: path + 'micro-frontends',
+                title: 'Get Started',
                 children: [
                     {
-                        title: 'React',
-                        path: path  + 'micro-frontends/react.md',
-                    },
-                    {
-                        title: 'Angular',
-                        path: path  + 'micro-frontends/angular.md',
-                    },
-                    {
-                        title: 'Communication',
-                        path: path  + 'micro-frontends/communication.md',
-                    },
-                    {
-                        title: 'Config',
-                        path: path  + 'micro-frontends/widget-configuration.md',
-                    },
-                    {
-                        title: 'Authentication',
-                        path: path  + 'micro-frontends/authentication.md',
-                    },
+                        title: 'Tutorials Learning Paths',
+                        path: path, 
+                    },
+                    {
+                        title: 'Install Entando',
+                        children: [
+                            {
+                                title: 'On Red Hat OpenShift',
```

Comment:
As we discussed let's drop the "On" in each one, so we'll get "Red Hat OpenShift", "Red Hat OpenShift using the OperatorHub", etc.